doc/nrf/libraries/others/supl_os_client.rst

```diff
@@ -102,8 +102,7 @@ See :ref:`configure_application` for information on how to change configuration
 
 .. note::
 
-   The SUPL client library requires that the newlib C library is used.
-   You can enable it using the :kconfig:option:`CONFIG_NEWLIB_LIBC` Kconfig option.
+   SUPL client library v0.8.0 or later is required when picolibc is used.
```

Comment:
```suggestion
   SUPL client library v0.8.0 or later is required when Picolibc is used.
```

Reply:
picolibc is an implementation of the C standard library, libc. Usually the names of these libraries are in lower case. There are other implementations which are also lower case, like glibc and newlib.

Reply:
After discussion offline, kept the current naming.